src/middlewared/middlewared/api/v25_04_0/smartctl.py

```diff
@@ -10,6 +10,7 @@ class AtaSelfTest(BaseModel):
     status_verbose: str
     remaining: float
     lifetime: int
+    poh_ago: int # current lifetime - lifetime at test start, "power on hours ago"
```

Comment:
Can we use a complete name? `powered_on_hours_ago` is not a super long identifier (just 6 chars longer than `status_verbose`) and is self-explanatory. `poh` is a term that we'll need to look up in the documentation every time we see it.